src/System.Private.ServiceModel/tests/Scenarios/Contract/XmlSerializer/XmlSerializerFormatTest.cs

```diff
@@ -69,4 +69,71 @@ public static void XmlSerializerFormatAttribute_SupportFaults()
             ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
         }
     }
+
+    [WcfFact]
+    [OuterLoop]
+    public static void XmlSFAttributeWsdlTest()
+    {
+        RunVariation(Endpoints.BasciHttpRpcEncSingleNs_Address);
+        RunVariation(Endpoints.BasicHttpRpcLitSingleNs_Address);
+        RunVariation(Endpoints.BasicHttpDocLitSingleNs_Address);
+        RunVariation(Endpoints.BasicHttpRpcEncMultiNs_Address, true);
+        RunVariation(Endpoints.BasicHttpRpcLitMultiNs_Address, true);
+        RunVariation(Endpoints.BasicHttpDocLitMultiNs_Address, true);
```

Comment:
Can we split this into 6 tests? It would be easier to know which one failed if any of them failed.

Reply:
Split into 6 tests.